spec/rubocop/cop/style/redundant_line_continuation_spec.rb

```diff
@@ -642,6 +642,101 @@ def do_something
     RUBY
   end
 
+  it 'registers an offense for a redundant continuation in a `begin` following a required continuation' do
```

Comment:
The `begin` keyword does not appear to be used in this test. It is likely referring to the "begin node," but as a spec, a different expression might make it clearer. For example, how about framing it around line continuations in the context of `do...end` blocks and the presence or absence of parentheses in method arguments?